reference/replica-rebuild.md

```diff
@@ -0,0 +1,10 @@
+## Replica Rebuilds
+
+With the previous versions, the control plane ensured replica redundancy by monitoring all nexuses and checking for any nexus that were in `Degraded` state, indicating that one or more replicas of that nexus were faulty. When a matching nexus is found, the faulty replica is removed. Then, a new replica is created and added to the nexus object. As part of adding the new child data-plane, a full rebuild was initiated from one of the existing `Online` replicas.
```

Comment:
let's replace the word nexus with `volume targets` as users won't know what a nexus is.